tests/test_doctestplus.py

```diff
@@ -730,7 +732,13 @@ def f():
     testdir.makefile('.rst', foo='>>> 1+1\n2')
 
     testdir.inline_run('--doctest-plus').assertoutcome(passed=2)
-    testdir.inline_run('--doctest-plus', '--doctest-rst').assertoutcome(passed=3)
+    if os.name == "nt" and python_version() == "3.14.0" and not PYTEST_LT_8_5:
+        with warnings.catch_warnings():
+            # ResourceWarning unclosed file pytest.EXE --> PytestUnraisableExceptionWarning
+            warnings.filterwarnings("ignore")
```

Comment:
this may be a bit too wide, but there are so many conditions it needs to catch that I don't mind it much.